samples/system-test/recognize.test.js

```diff
@@ -13,6 +13,8 @@
  * limitations under the License.
  */
 
+/* eslint-disable */
```

Comment:
Why is this being added to an existing file?

Please explain this.

I will leave a number of these comments. On one of them, please describe why this is necessary & what workarounds you have tried to fix this before disabling eslint entirely in these files. Thank you.